crates/chain/tests/test_tx_graph.rs

```diff
@@ -88,57 +88,21 @@ fn insert_txouts() {
 
     // Make the update graph
     let update = {
-        let mut graph = TxGraph::default();
+        let mut update = tx_graph::Update::default();
         for (outpoint, txout) in &update_ops {
-            // Insert partials transactions
-            assert_eq!(
-                graph.insert_txout(*outpoint, txout.clone()),
-                ChangeSet {
-                    txouts: [(*outpoint, txout.clone())].into(),
-                    ..Default::default()
-                }
-            );
```

Comment:
note sure if it's worth it but you could keep all the code here the same and then just turn the `TxGraph` into an `Update` at the end. Might as well to test the `From` impl but up to you.

Reply:
I think it's better to add separate tests for those!

Reply:
Tests added.